src/mono/wasm/Makefile

```diff
@@ -91,6 +92,28 @@ build-interp-$(1): $(BUILDS_BIN_DIR)/$(1)/dotnet.js
 
 endef
 
+# Needed by AOT
+
+SRC_FILES=\
+	runtime/driver.c \
+	runtime/corebindings.c \
+	runtime/binding_support.js \
+	runtime/dotnet_support.js \
+	runtime/library_mono.js
+
+DST_FILES=$(patsubst runtime/%,$(BUILDS_SRC_DIR)/%, $(SRC_FILES))
```

Comment:
I don't think this is the best approach, we'll have most likely different runtime pack for aot configuration(s).

Reply:
What will be in those packs ?

Reply:
For example different code because SRE won't be supported, AOT specific binding, wasm version of VM, etc